### PR TITLE
fix csproj relative paths, smartstore sqlite rtm extension, assemblyinfo bug

### PR DESCRIPTION
**Pull request by wpeter-sfdc**
@jhsfdc @smcnulty-sfdc @kchitalia

### CODE REVIEW

**Comment by wpeter-sfdc** [comment]
don't merge yet, one more fix coming

**Comment by wpeter-sfdc** [comment]
weird, github merged it but I didnt' tell it to... when I pushed the last commit it just auto merged... anyway, please review this PR and then also the last commit - https://github.com/jhsfdc/SalesforceMobileSDK-Windows/commit/8d7cef6e64c7b59d318f4df739b15ca7adf38fd6

File: SalesforceSDK/Hybrid.SmartSync/Hybrid.SmartSync.csproj [hunk 1]
```diff
@@ -124,7 +124,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
```
Comment (smcnulty-sfdc) [inline SalesforceSDK/Hybrid.SmartSync/Hybrid.SmartSync.csproj]: Should be version 7.0.1

File: SalesforceSDK/Hybrid/Hybrid.csproj [hunk 2]
```diff
@@ -126,7 +126,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
```
Comment (smcnulty-sfdc) [inline SalesforceSDK/Hybrid/Hybrid.csproj]: ditto... 7.0.1

File: SalesforceSDK/Salesforce.SDK.Tests/Salesforce.SDK.Tests.csproj [hunk 3]
```diff
@@ -174,11 +174,11 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Newtonsoft.Json, Version=6.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
+      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.5\lib\netcore45\Newtonsoft.Json.dll</HintPath>
```
Comment (smcnulty-sfdc) [inline SalesforceSDK/Salesforce.SDK.Tests/Salesforce.SDK.Tests.csproj]: 7.0.1

File: SalesforceSDK/Universal/Universal.csproj [hunk 4]
```diff
@@ -154,7 +154,7 @@
   </ItemGroup>
   <ItemGroup>
     <Reference Include="Newtonsoft.Json">
-      <HintPath>$(SolutionDir)\packages\Newtonsoft.Json.6.0.5\lib\portable-net45+wp80+win8+wpa81\Newtonsoft.Json.dll</HintPath>
+      <HintPath>$(SolutionDir)packages\Newtonsoft.Json.6.0.5\lib\portable-net45+wp80+win8+wpa81\Newtonsoft.Json.dll</HintPath>
```
Comment (smcnulty-sfdc) [inline SalesforceSDK/Universal/Universal.csproj]: 7.0.1

**Comment by wpeter-sfdc** [comment]
Right, the PR missed my last commit, if you look at it you'll see I fixed all the json.net references - that commit is at https://github.com/jhsfdc/SalesforceMobileSDK-Windows/commit/8d7cef6e64c7b59d318f4df739b15ca7adf38fd6

**Comment by jbernoudy** [comment]
:+1:

**Pull request merged**